<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ToolsGeneralWidget</class>
 <widget class="QWidget" name="ToolsGeneralWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>833</width>
    <height>437</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QWidget" name="panelEditButtons" native="true">
     <layout class="QHBoxLayout" name="horizontalLayout_5">
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QToolButton" name="btnAdd">
        <property name="text">
         <string>Add</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QToolButton" name="btnEdit">
        <property name="text">
         <string>Edit</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QToolButton" name="btnRemove">
        <property name="text">
         <string>Remove</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QWidget" name="widget" native="true">
     <layout class="QHBoxLayout" name="horizontalLayout">
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QWidget" name="widget_5" native="true">
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QListView" name="lstTools">
        <property name="editTriggers">
         <set>QAbstractItemView::NoEditTriggers</set>
        </property>
        <property name="dragEnabled">
         <bool>true</bool>
        </property>
        <property name="dragDropMode">
         <enum>QAbstractItemView::InternalMove</enum>
        </property>
        <property name="defaultDropAction">
         <enum>Qt::MoveAction</enum>
        </property>
        <property name="alternatingRowColors">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QWidget" name="panelEdit" native="true">
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <property name="leftMargin">
          <number>0</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <item>
          <widget class="QFrame" name="frmEdit">
           <property name="frameShape">
            <enum>QFrame::StyledPanel</enum>
           </property>
           <property name="frameShadow">
            <enum>QFrame::Raised</enum>
           </property>
           <layout class="QGridLayout" name="gridLayout">
            <item row="1" column="2">
             <widget class="QToolButton" name="btnBrowseProgram">
              <property name="text">
               <string>Browse</string>
              </property>
             </widget>
            </item>
            <item row="2" column="2">
             <widget class="QToolButton" name="btnBrowseWorkingDirectory">
              <property name="text">
               <string>Browse</string>
              </property>
             </widget>
            </item>
            <item row="6" column="1" colspan="2">
             <widget class="QComboBox" name="cbOutput"/>
            </item>
            <item row="1" column="0">
             <widget class="QLabel" name="label_2">
              <property name="text">
               <string>Program</string>
              </property>
             </widget>
            </item>
            <item row="6" column="0">
             <widget class="QLabel" name="label_6">
              <property name="text">
               <string>Output To</string>
              </property>
             </widget>
            </item>
            <item row="1" column="1">
             <widget class="QLineEdit" name="txtProgram"/>
            </item>
            <item row="3" column="0">
             <widget class="QLabel" name="label_4">
              <property name="text">
               <string>Parameters</string>
              </property>
             </widget>
            </item>
            <item row="11" column="0">
             <spacer name="verticalSpacer">
              <property name="orientation">
               <enum>Qt::Vertical</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>20</width>
                <height>40</height>
               </size>
              </property>
             </spacer>
            </item>
            <item row="5" column="0">
             <widget class="QLabel" name="label_5">
              <property name="text">
               <string>Redirect Input</string>
              </property>
             </widget>
            </item>
            <item row="0" column="1" colspan="2">
             <widget class="QLineEdit" name="txtTitle"/>
            </item>
            <item row="2" column="0">
             <widget class="QLabel" name="label_3">
              <property name="text">
               <string>Working Directory</string>
              </property>
             </widget>
            </item>
            <item row="3" column="1" colspan="2">
             <widget class="QLineEdit" name="txtParameters"/>
            </item>
            <item row="0" column="0">
             <widget class="QLabel" name="label">
              <property name="text">
               <string>Title</string>
              </property>
             </widget>
            </item>
            <item row="8" column="0" colspan="3">
             <widget class="Line" name="line">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
             </widget>
            </item>
            <item row="10" column="0" colspan="3">
             <widget class="QWidget" name="widget_2" native="true">
              <layout class="QHBoxLayout" name="horizontalLayout_4">
               <property name="leftMargin">
                <number>0</number>
               </property>
               <property name="topMargin">
                <number>0</number>
               </property>
               <property name="rightMargin">
                <number>0</number>
               </property>
               <property name="bottomMargin">
                <number>0</number>
               </property>
               <item>
                <widget class="QPushButton" name="btnInsertMacro">
                 <property name="text">
                  <string>Insert Macro</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QComboBox" name="cbMacros">
                 <property name="sizePolicy">
                  <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
                   <horstretch>0</horstretch>
                   <verstretch>0</verstretch>
                  </sizepolicy>
                 </property>
                 <property name="editable">
                  <bool>false</bool>
                 </property>
                 <property name="sizeAdjustPolicy">
                  <enum>QComboBox::AdjustToContents</enum>
                 </property>
                </widget>
               </item>
              </layout>
             </widget>
            </item>
            <item row="12" column="0" colspan="3">
             <widget class="QWidget" name="widget_3" native="true">
              <property name="minimumSize">
               <size>
                <width>0</width>
                <height>0</height>
               </size>
              </property>
              <layout class="QHBoxLayout" name="horizontalLayout_3">
               <property name="leftMargin">
                <number>0</number>
               </property>
               <property name="topMargin">
                <number>0</number>
               </property>
               <property name="rightMargin">
                <number>0</number>
               </property>
               <property name="bottomMargin">
                <number>0</number>
               </property>
               <item>
                <spacer name="horizontalSpacer_2">
                 <property name="orientation">
                  <enum>Qt::Horizontal</enum>
                 </property>
                 <property name="sizeHint" stdset="0">
                  <size>
                   <width>40</width>
                   <height>20</height>
                  </size>
                 </property>
                </spacer>
               </item>
               <item>
                <widget class="QPushButton" name="btnEditOk">
                 <property name="text">
                  <string>Ok</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QPushButton" name="btnEditCancel">
                 <property name="text">
                  <string>Cancel</string>
                 </property>
                </widget>
               </item>
              </layout>
             </widget>
            </item>
            <item row="2" column="1">
             <widget class="QLineEdit" name="txtDirectory"/>
            </item>
            <item row="4" column="0" colspan="3">
             <widget class="QLineEdit" name="txtDemo">
              <property name="readOnly">
               <bool>true</bool>
              </property>
              <property name="clearButtonEnabled">
               <bool>false</bool>
              </property>
             </widget>
            </item>
            <item row="5" column="1" colspan="2">
             <widget class="QComboBox" name="cbInput"/>
            </item>
            <item row="7" column="0" colspan="3">
             <widget class="QCheckBox" name="chkUTF8">
              <property name="text">
               <string>Use UTF8 as the encoding</string>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>btnAdd</tabstop>
  <tabstop>btnEdit</tabstop>
  <tabstop>btnRemove</tabstop>
  <tabstop>lstTools</tabstop>
  <tabstop>txtTitle</tabstop>
  <tabstop>txtProgram</tabstop>
  <tabstop>btnBrowseProgram</tabstop>
  <tabstop>txtDirectory</tabstop>
  <tabstop>btnBrowseWorkingDirectory</tabstop>
  <tabstop>txtParameters</tabstop>
  <tabstop>txtDemo</tabstop>
  <tabstop>cbInput</tabstop>
  <tabstop>cbOutput</tabstop>
  <tabstop>chkUTF8</tabstop>
  <tabstop>btnInsertMacro</tabstop>
  <tabstop>cbMacros</tabstop>
  <tabstop>btnEditOk</tabstop>
  <tabstop>btnEditCancel</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
